src/store/modules/ResourceManagement/SystemParametersStore.js

```diff
@@ -47,6 +84,148 @@ const systemParametersStore = {
         })
         .catch((error) => console.log(error));
     },
+    async getRpdPolicy({ commit }) {
+      return await api
+        .get(
+          '/redfish/v1/Registries/BiosAttributeRegistry/BiosAttributeRegistry'
+        )
+        .then(({ data: { RegistryEntries } }) => {
+          const rpdPolicy = RegistryEntries.Attributes.filter(
+            (Attribute) => Attribute.AttributeName == 'pvm_rpd_policy'
+          );
+          let rpdPolicyValue = rpdPolicy[0].CurrentValue;
+          commit('setRpdPolicy', rpdPolicyValue);
+          commit('setPvmRpdPolicy', rpdPolicyValue);
+        })
+        .catch((error) => console.log(error));
+    },
+    async getRpdPolicyCurrent({ commit }) {
+      return await api
+        .get(
+          '/redfish/v1/Registries/BiosAttributeRegistry/BiosAttributeRegistry'
+        )
+        .then(({ data: { RegistryEntries } }) => {
+          const rpdPolicyCurr = RegistryEntries.Attributes.filter(
+            (Attribute) => Attribute.AttributeName == 'pvm_rpd_feature_current'
+          );
+          let rpdPolicyCurrValue = rpdPolicyCurr[0].CurrentValue;
+          commit('setRpdPolicyCurrent', rpdPolicyCurrValue);
+        })
+        .catch((error) => console.log(error));
+    },
+    async getRpdFeature({ commit }) {
+      return await api
+        .get(
+          '/redfish/v1/Registries/BiosAttributeRegistry/BiosAttributeRegistry'
+        )
+        .then(({ data: { RegistryEntries } }) => {
+          const rpdFeature = RegistryEntries.Attributes.filter(
+            (Attribute) => Attribute.AttributeName == 'pvm_rpd_feature_current'
```

Comment:
I think we have to use **pvm_rpd_feature** instead of **pvm_rpd_feature_current**.